ionos_dev/search-engine/search/deployment.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: dome-search-deployment
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/instance: dome-search
+      app.kubernetes.io/name: dome-search
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/instance: dome-search
+        app.kubernetes.io/name: dome-search
+    spec:
+      containers:
```

Comment:
No health and readiness checks are configured, no option to set resource-restricitons is provided. I think that needs to be changed .

Reply:
At the moment no readness/health check is provided for the search-engine. It will provide asap

Reply:
We have added the livenessProbe for the search software